guides/Usage.md

```diff
@@ -116,6 +116,35 @@ project %ItemUpdated{uuid: uuid} = event, _metadata, fn multi ->
 end
 ```
 
+
+### Using the `project_batch` macro
+
+By configuring the base event handler to use batching and using the `project_batch/2` macro, you can project a batch of events in a single step within a transaction. `project_batch/2` receives a list of `{event, metadata}` tuples for all the events in the batch and a similar single-arity function as `project/3` to affect an `Ecto.Multi` structure.
+
+Note that there is currently no built in way to target a single type of event to be projected, and as such a single `project_batch` macro is expected to gracefully handle (or ignore) any events that it may receive
+
+#### Example
+```elixir
+defmodule MyApp.Projections.BatchProjector do
+  use Commanded.Projections.Ecto,
+      application: MyApp.Application,
+      repo: MyApp.Projections.Repo,
+      name: "example_batch_projection",
+      callback_handler: :batch, # This tells the underlying EventHandler to batch events
```

Comment:
minor, but I would expect something called `callback_handler` to be a module and this is a setting. `callback_handler_mode`?

Reply:
The docs is outdated. In order for batching to be used we need 
* set batch_size
* using `project_batch/2`
Will update